src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -49,21 +49,45 @@ class AdminModuleDataProvider implements ModuleInterface
      * @const array giving a translation domain key for each module action
      */
     public const _ACTIONS_TRANSLATION_DOMAINS_ = [
-        'install' => 'Admin.Actions',
-        'uninstall' => 'Admin.Actions',
-        'enable' => 'Admin.Actions',
-        'disable' => 'Admin.Actions',
-        'enableMobile' => 'Admin.Modules.Feature',
-        'disableMobile' => 'Admin.Modules.Feature',
-        'reset' => 'Admin.Actions',
-        'upgrade' => 'Admin.Actions',
+        Module::ACTION_INSTALL => 'Admin.Actions',
+        Module::ACTION_UNINSTALL => 'Admin.Actions',
+        Module::ACTION_ENABLE => 'Admin.Actions',
+        Module::ACTION_DISABLE => 'Admin.Actions',
+        Module::ACTION_ENABLE_MOBILE => 'Admin.Modules.Feature',
+        Module::ACTION_DISABLE_MOBILE => 'Admin.Modules.Feature',
+        Module::ACTION_RESET => 'Admin.Actions',
+        Module::ACTION_UPGRADE => 'Admin.Actions',
         'configure' => 'Admin.Actions',
     ];
 
+    /**
+     * @const array giving a translation label for each module action
+     */
+    public const _ACTIONS_TRANSLATION_LABELS_ = [
+        Module::ACTION_INSTALL => Module::ACTION_INSTALL,
+        Module::ACTION_UNINSTALL => Module::ACTION_UNINSTALL,
+        Module::ACTION_ENABLE => Module::ACTION_ENABLE,
+        Module::ACTION_DISABLE => Module::ACTION_DISABLE,
+        Module::ACTION_ENABLE_MOBILE => 'Enable mobile',
+        Module::ACTION_DISABLE_MOBILE => 'Disable mobile',
+        Module::ACTION_RESET => Module::ACTION_RESET,
+        Module::ACTION_UPGRADE => Module::ACTION_UPGRADE,
+        'configure' => 'configure',
+    ];
```

Comment:
I don't think it is a good idea to use action name as label. 
If action name change Ex.: install -> installModule the action label will be automatically wrong

Reply:
Fixed. Thanks @PululuK